projects/envoy/Dockerfile

```diff
@@ -27,12 +27,9 @@ RUN apt-get update && apt-get -y install  \
     libtool         \
     wget            \
     golang          \
-    python	    \
     rsync
 
-# Install Bazelisk
-RUN wget -O /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v0.0.8/bazelisk-linux-amd64
-RUN chmod +x /usr/local/bin/bazel
+RUN pip3 install six
```

Comment:
Will six be needed just for envoy or all projects ?

Reply:
Good point - it is actually required by the Bazel rules for fuzzing. I've added it instead to the base image Dockerfile, PTAL.